resources/attributes/default.rb

```diff
@@ -92,6 +93,8 @@
 default["redborder"]["memory_services"]["n2klocd"] = {"count" => 10, "memory" => 0 }
 default["redborder"]["memory_services"]["redborder-cep"] = {"count" => 10, "memory" => 0 }
 default["redborder"]["memory_services"]["rb-aioutliers"] = {"count" => 10, "memory" => 0 }
+# excluded mem services
+default['redborder']['excluded_memservices'] = Set.new(['chef-client']) # Don't assign memory to chef because the service will get handled
```

Comment:
Why using Set here?

Since you can create an `Array` and do `include?` also

`services = ["chef-client"]
puts services.include? "chef-client" => True`

Reply:
default['redborder']['excluded_memservices'] = %w[chef-client]